src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Views/BankId/_LoginScript.cshtml

```diff
@@ -116,6 +120,12 @@
                         if (data.checkStatus) {
                             checkStatus(requestVerificationToken, returnUrl, loginOptions, data.orderRef);
                         }
+
+                        cancelButtonElement.onclick = function(e) {
```

Comment:
Use `addEventListener` for listetning to JS events, like this:

```
            cancelButtonElement.addEventListener('click', function (event) {
                ......
            });
```

Reply:
Ok!